src/App/Pages/Accounts/EnvironmentPageViewModel.cs

```diff
@@ -22,10 +23,10 @@ public EnvironmentPageViewModel()
             IdentityUrl = _environmentService.IdentityUrl;
             IconsUrl = _environmentService.IconsUrl;
             NotificationsUrls = _environmentService.NotificationsUrl;
-            SubmitCommand = new Command(async () => await SubmitAsync());
+            SubmitCommand = new AsyncCommand(SubmitAsync, allowsMultipleExecutions: false);
```

Comment:
Here for the `SubmitAsync` we need to add some kind of exception handling, at least to log the exception to AppCenter in case that happens; you can use `ILogger` for that and the parameter of `AsyncCommand` -> `onException:` (check example [here](https://github.com/bitwarden/mobile/blob/3aef86bd3402f21d2f14249dfa04f5f5e3a8e04a/src/App/Controls/AccountSwitchingOverlay/AccountSwitchingOverlayViewModel.cs#L26))
Also you can display a generic error to the user after logging it.